src/lib/util/options.ml

```diff
@@ -545,7 +545,7 @@ let mk_sat_opt bottom_classes disable_flat_formulas_simplification
          match s with
          | "" -> set
          | s -> Util.SS.add s set
-      ) Util.SS.empty (Str.split (Str.regexp ",") ndo)
```

Comment:
Note that `Str` is now supported by js_of_ocaml.3.6

Reply:
Good, this could be usefull